package.json

```diff
@@ -32,51 +32,23 @@
   },
   "homepage": "https://github.com/arawa/workspace#readme",
   "devDependencies": {
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "^7.16.11",
+    "@nextcloud/babel-config": "^1.0.0",
     "@nextcloud/browserslist-config": "^2.2.0",
-    "@nextcloud/eslint-config": "^4.0.0",
-    "@nextcloud/eslint-plugin": "^2.0.0",
-    "@nextcloud/webpack-vue-config": "^3.0.0",
+    "@nextcloud/eslint-config": "^6.1.2",
+    "@nextcloud/stylelint-config": "^2.1.2",
+    "@nextcloud/webpack-vue-config": "^4.3.2",
     "@vue/test-utils": "^1.3.0",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.2.3",
-    "chai": "^4.3.4",
-    "css-loader": "^5.2.7",
-    "eslint-webpack-plugin": "^2.6.0",
-    "jsdom": "^19.0.0",
+    "chai": "^4.3.6",
     "jsdom-global": "^3.0.2",
-    "mocha": "^9.2.0",
-    "mochapack": "^2.1.2",
-    "postcss-html": "^1.3.0",
-    "prettier": "^2.5.1",
-    "stylelint": "^14.3.0",
-    "stylelint-config-recommended-scss": "^4.2.0",
-    "stylelint-config-recommended-vue": "^1.1.0",
-    "stylelint-scss": "^3.21.0",
-    "url-loader": "^4.1.1",
-    "vue-loader": "^15.9.6",
-    "vue-template-compiler": "^2.6.12",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^4.9.2",
-    "webpack-merge": "^5.7.3",
+    "mochapack": "^2.1.4",
     "webpack-node-externals": "^3.0.0"
   },
   "dependencies": {
-    "@babel/core": "^7.16.12",
-    "@babel/eslint-parser": "^7.16.5",
     "@nextcloud/axios": "^1.9.0",
     "@nextcloud/l10n": "^1.4.1",
     "@nextcloud/router": "^1.2.0",
     "@nextcloud/vue": "^4.3.0",
-    "eslint": "^7.23.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.3.1",
-    "eslint-plugin-vue": "^7.8.0",
```

Comment:
This is where your issue is @zak39 :)
You have conflicts with `eslint` because you have some in `dependencies`, but they should have been in `devDependencies`

Reply:
Aaaah yes !!!

I am stupid !!! ><

I am fixing my release npm and I rebuild again to see that !